test/e2e/helpers.js

```diff
@@ -120,8 +120,28 @@ async function withFixtures(options, testSuite) {
       await driver.checkBrowserForExceptions();
     }
 
+    let driverProxy;
+    if (process.env.E2E_DEBUG === 'true') {
+      driverProxy = new Proxy(driver, {
+        get(target, prop, receiver) {
+          const originalProperty = target[prop];
+          if (typeof originalProperty === 'function') {
+            return (...args) => {
+              console.log(
+                `[driver] Called '${prop}' with arguments '${JSON.stringify(
+                  args,
+                )}'`,
```

Comment:
Nit: I'm wondering if the single quotes around the arguments could be slightly confusing. What do you think about:

``` suggestion
                `[driver] Called '${prop}' with arguments ${JSON.stringify(
                  args,
                )}`,
```

Reply:
Could you elaborate on why it might be confusing?

As a general rule, I always like to quote values embedded in debug logs, so that it's more clear where the message stops and the value begins. It can help validate cases where the value is an empty string as well for example.

Reply:
Well, I guess in this case it is guaranteed to at least be an array. I hadn't considered that. The square brackets are an adequate delimiter. I'll remove these quotes.

Reply:
Someone who is not familiar with what the mechanics of the e2e tests might interpret this message as meaning that the function is being called with a string rather than a list of values. I see what you're saying though. We could consider using a different character (e.g. `«`/`»`). I guess it's not a deal until it is :)